packages/react-icons/convert.js

```diff
@@ -155,7 +151,6 @@ export const ${destFilename} = /*#__PURE__*/wrapIcon(/*#__PURE__*/${destFilename
 
   for(const chunk of iconChunks) {
     chunk.unshift(`import wrapIcon from "../utils/wrapIcon";`)
-    chunk.unshift(`import { FluentIconsProps } from "../utils/FluentIconsProps.types";`)
```

Comment:
If removing this, does the FluentIconsProps.types file also need to be removed, or at least the typedef?

Reply:
The `FluentIconsProps.types` still needs to be defined, it just doesn't need to be used to define `className` and `primaryFill` props anymore, thus is unnecessary for the chunk files